src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -55,4 +74,15 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName, @Reque
         }
     }
 
+    private Optional<ValidationError> validateSchema(@RequestBody String event, EventType eventType) {
```

Comment:
Refactoring tool did it automatically. Fixed here https://github.com/zalando/nakadi/commit/969a3635cfcf53b6c40bbde008986b5135cd8b46